packages/babel-compat-data/scripts/data/plugin-features.js

```diff
@@ -91,7 +108,12 @@ const es2015 = {
     features: ["Symbol / typeof support"],
   },
   "transform-new-target": {
-    features: ["new.target"],
+    features: [
+      "new.target",
+      'arrow functions / lexical "super" binding in constructors',
+      'arrow functions / lexical "super" binding in methods',
+      'arrow functions / lexical "new.target" binding',
```

Comment:
And we only need the `new.target` on here.

Reply:
Ah! Looking at it with that in mind it was sort of obvious haha